src/metis/adapter.c

```diff
@@ -129,11 +129,8 @@ void galahad_metis5_adapter(idx_t* nvtxs, idx_t* xadj, idx_t* adjncy,
         options5[METIS_OPTION_PFACTOR] = options[6];
 
         options5[METIS_OPTION_NSEPS] = options[7];
-    }
-
-    /* override inappropriate MeTiS 5 ufactor option */
 
-    options5[METIS_OPTION_UFACTOR] = 1;
```

Comment:
@nimgould @jfowkes Are you sure to want this modification?

Reply:
THere were many examples for which making this change reduced the times by a factor of 10-100 without really changing the quality of the permutation, particularly for larger examples.